tests/mesolve/mesolver_tester.py

```diff
@@ -72,3 +72,46 @@ def _test_rho_save(
 
     def test_rho_save(self):
         pass
+
+
+class AdjointMESolverTester(MESolverTester):
```

Comment:
```suggestion
class MEAdjointSolverTester(MESolverTester):
```
It's a _Master Equation Adjoint Solver_ and not an _Adjoint Master Equation Solver_.